src/main/java/com/hillel/items_exchange/controller/AuthController.java

```diff
@@ -43,20 +52,23 @@
     private final JwtTokenProvider jwtTokenProvider;
 
     @PostMapping("/login")
-    public ResponseEntity< Map<String, String>> login(@RequestBody @Valid UserLoginDto userLoginDto, BindingResult bindingResult) {
+    public ResponseEntity<Map<String, String>> login(@RequestBody @Valid UserLoginDto userLoginDto,
+                                                     BindingResult bindingResult) {
+
         userLoginDtoValidator.validate(userLoginDto, bindingResult);
         if (bindingResult.hasErrors()) {
             throw new ResourceNotFoundException(bindingResult.toString());
         }
 
         try {
             String username = userLoginDto.getUsernameOrEmail();
-            authenticationManager.authenticate(new UsernamePasswordAuthenticationToken(username, userLoginDto.getPassword()));
+            authenticationManager.authenticate(new UsernamePasswordAuthenticationToken(username,
+                    userLoginDto.getPassword()));
+
             User user = userService.findByUsernameOrEmail(username)
                     .orElseThrow(() -> new UsernameNotFoundException("User with username: " + username + " not found"));
```

Comment:
Would you mind replace message in exception to message from properties?

Reply:
Ok